gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/AliasBasedTokenStateService.java

```diff
@@ -125,11 +126,27 @@ protected boolean isUnknown(final String token) {
     return isUnknown;
   }
 
+  @Override
+  protected void removeRevokedExpiredToken(final String token) {
```

Comment:
I think the more simply-named removeToken(String) would be better. Who cares why it's being removed?

Reply:
Addressed in new update.